projects/valtimo/dossier/src/lib/components/dossier-detail-task-list/dossier-detail-task-list.component.html

```diff
@@ -0,0 +1,76 @@
+<!--
+  ~ Copyright 2015-2024 Ritense BV, the Netherlands.
+  ~
+  ~ Licensed under EUPL, Version 1.2 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" basis,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<div class="task-list" *ngIf="{loadingTasks: loadingTasks$ | async} as obs">
+  <h2 class="task-list__title">{{ 'summary.userTasks' | translate }}</h2>
+
+  <div
+    class="task-list__tasks-container"
+    cdsLayer="1"
+    *ngIf="!obs.loadingTasks; else loadingTasksTemplate"
+  >
+    @if (!(tasks || []).length) {
+      <ng-container *ngTemplateOutlet="noTasksTemplate"></ng-container>
+    }
+
+    @for (task of tasks || []; track task.id) {
+      <ng-container *ngTemplateOutlet="taskTemplate; context: {task: task}"></ng-container>
+    }
+  </div>
+
+  <valtimo-task-detail-modal
+    #taskDetail
+    (formSubmit)="init()"
+    (assignmentOfTaskChanged)="init()"
+  ></valtimo-task-detail-modal>
+</div>
+
+<ng-template #taskTemplate let-task="task">
+  <cds-clickable-tile (click)="rowTaskClick(task)">
+    <div class="task">
```

Comment:
Is this div necessary?

Reply:
Yes, otherwise I have to target cds-tile with ::ng-deep which I'd like to avoid